src/vs/platform/clipboard/browser/clipboardService.ts

```diff
@@ -98,3 +160,5 @@ export class BrowserClipboardService implements IClipboardService {
 		return this.resources.length > 0;
 	}
 }
+
+registerSingleton(IClipboardService, BrowserClipboardService, true);
```

Comment:
This is more my personal preference than written somewhere: but I prefer to not self register services in `platform` simply because whoever depends on this service might have their own way of registering services. As such, please make it similar to the other platform services in `wokbench.web.main.ts` and register it there:

https://github.com/microsoft/vscode/blob/87bf1d80652c928d4c81cfa95b05bbf285e6dd84/src/vs/workbench/workbench.web.main.ts#L90